addons/pos_restaurant/models/pos_config.py

```diff
@@ -42,3 +42,12 @@ def _onchange_is_table_management(self):
     def _onchange_is_order_printer(self):
         if not self.is_order_printer:
             self.printer_ids = [(5, 0, 0)]
+
+    @api.model
+    def get_tables_order_count(self, config_id):
```

Comment:
you can set it @api.multi, and if you give the correct_id, you won't need to browse